src/views/PeopleSearch/components/SearchFieldList/components/SearchField/index.tsx

```diff
@@ -18,6 +18,7 @@ interface CommonProps {
   defaultDisabled?: boolean;
   select?: boolean;
   options?: string[] | SelectOption[];
+  allIndicator?: boolean;
```

Comment:
The problem we're trying to solve right now is that we don't want an 'All' rather we want an option between 'All' and 'None' as default values. (I've yet to take a look at the PR yet)

Reply:
Right, I understand that. I'm proposing that it would be both clearer and easier to use in the future if instead of making it an either choice, you allow the component's consumer to choose between the default label of 'All' and any other label they want to provide.